content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,198 @@
+
+Datadog also has an [APM integration with Synthetics][6] which allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use * for wildcards: `https://*.datadoghq.com/*`
+
+Alerts triggered with the APM integration populate under **Test Results**.
+
+## Create an API test
+
+[API tests][7] help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as response time, HTTP status code, and header or body contents. Use the [Datadog API][8] to see the full list.
+
+In this example, an API test is created to ensure your website is constantly up and providing responses in a specific amount of time.
+
+### Configure the request
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test:
+    - Add a start URL.
+    - Select **Advance Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - Add one or two tags, like `prod`, to help organize and filter tests.
+    - Select locations for testing.
+    - Click the **Test URL** button.
+
+#### Define your alert conditions
+
+Now that the test is completed, if you did not create any [assertions][9], they are automatically populated. API tests require at least one assertion to be monitored by Datadog. An assertion is defined by a parameter, an optional property, a comparator, and a target value.
+
+In this example, three default assertions are populated when testing the URL:
+```
+When status code is 200
+And header content-type is text/html; charset=utf
+And response time less than 2000 ms
+```
+
+These assertions define the alert condition and can be customized. To add a custom assertion, click on any response header in the response preview. You can also click the **New Assertion** button to add an assertion manually, (e.g., `body` contains `Shop.ist`.)
+
+To configure the manual retry of a test when it fails, set:
+```
+An alert is triggered if your test fails for x minutes from any x of x locations`
+Retry x time before location is marked as failed
+```
+
+Once alert conditions are set, create a message for the alert and specify what services and team members to receive the alert notification email and click **Save Test**. You can also use [integrations][10], such as Slack, Pagerduty, webhooks, etc., to receive alert notifications.
+
+{{< img src="getting_started/synthetics/api-test-configuration.gif" alt="Browser test failure" responsive="true" style="width:90%;">}}
```

Comment:
Can we move that right after the 
```
These assertions define the alert condition and can be customized. To add a custom assertion, click on any response header in the response preview. You can also click the **New Assertion** button to add an assertion manually, (e.g., `body` contains `Shop.ist`.)
```
paragraph?

Reply:
Moved to after this paragraph.